getting_started/first_2d_game/05.the_main_game_scene.rst

```diff
@@ -189,8 +173,6 @@ generator generates different random numbers each time the game is run:
         // Uncomment these after adding the nodes in the "Sound effects" section of "Finishing up".
         //_music = get_node<godot::AudioStreamPlayer>("Music");
         //_death_sound = get_node<godot::AudioStreamPlayer>("DeathSound");
-        _random = (godot::Ref<godot::RandomNumberGenerator>)godot::RandomNumberGenerator::_new();
-        _random->randomize();
```

Comment:
The `_random` field initialization shouldn't be removed, the `randomize()` method is fine to remove.